tests/Feature/QuestionApiTest.php

```diff
@@ -41,4 +42,15 @@ public function test_add_new_question_fail(): void
         ]);
         $response->assertValid(['message', 'errors']);
     }
+
+    public function test_delete_question_success(): void
+    {
+        $data = [Question::factory()->create()];
+        $response = $this->deleteJson('/api/questions/{id}');
```

Comment:
I'm not actually 100% sure this test is passing, but it does need an amend.

Get rid of the square brackets on line 48, and then instead of using `'/api/questions/{id}'` it should be `'/api/questions/' . $data->id`.

So using the factory to first make the question in the database, and then using the ID of that question in the delete request